### PR TITLE
feat: update es logo

### PR DESCRIPTION
**Pull request by hroth1994**
### 🔗 Linked issue

https://energysage.atlassian.net/browse/CED-1638

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

- Update to new logo

### 🥼 Testing

- Tested locally at http://localhost:8500/organisms/es-nav-bar and http://localhost:8500/organisms/es-footer

#### 🧐 Feedback Requested / Focus Areas

- General

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.
- [x] I have documented testing approach

### CODE REVIEW

**Review by ericdouglaspratt — APPROVED** [review]
Looks good but there are a lot of stylistic warnings in the diff - would be good to minimize those prior to merging.

File: es-design-system/components/ds-es-logo.vue [hunk 1]
```diff
@@ -1,56 +1,45 @@
 <template>
-    <svg
-        :style="{
-            height: height,
-            width: width,
```
Comment (nathanielwarner) [inline es-design-system/components/ds-es-logo.vue:5]: I don't think we should be removing the height and width passthrough here. It means the props will have no effect.

**Review by nathanielwarner — CHANGES_REQUESTED** [review]

File: es-design-system/components/ds-es-logo.vue (hunk 1 above)
Comment (hroth1994) [reply es-design-system/components/ds-es-logo.vue:5]: GREAT catch!

**Review by ericdouglaspratt — CHANGES_REQUESTED** [review]
For the footer, we should make sure the es-design-system layout page for the footer has this `class="text-white"` on the logo it's putting in the slot
https://github.com/EnergySage/es-cms-pages/blob/develop/layouts/default.vue#L19

**Review by ericdouglaspratt — APPROVED** [review]
Looks good! Tested nav and footer this time lol

**Review by nathanielwarner — APPROVED** [review]

**Pull request merged**